README.md

```diff
@@ -12,11 +12,19 @@ Add the library to your dependencies in your `build.gradle`:
 
 ```groovy
 dependencies {
-  compile 'com.braintreepayments:browser-switch:1.0.0'
+  implementation 'com.braintreepayments:browser-switch:1.0.0'
 }
 ```
 
-Declare `BrowserSwitchActivity` in your `AndroidManifest.xml`:
+To use the latest build from the `master` branch use:
+
+ ```groovy
+dependencies {
+  implementation 'com.braintreepayments:browser-switch:1.0.1-SNAPSHOT'
```

Comment:
This will become out-of-date as we release new versions, since we're specifying `1.0.1-SNAPSHOT`. Is there a way to tell gradle to always pull in the latest snapshot?

Reply:
Or does the Rakefile increment this somehow?

Reply:
Yeah this happens [here](https://github.com/braintree/browser-switch-android/blob/14b0bd23da08cd7636c9012eb6377f2736d9c0e7/Rakefile#L61). It would be nice to see if there are some official docs on naming SNAPSHOT artifacts. We advertise them in a lot of our repo's trying to make it more of a thing.

Reply:
Cool 👍

Reply:
My only concern here would be offering multiple options to integrate. Is it a common practice to use SNAPSHOTs? In the past I think we've only used them as one-offs for merchants to test betas or potential fixes.

Reply:
Yeah I understand that particular concern. Braintree core has advertised a snapshot for quite some time. We could eliminate snapshots across all projects that would make the README easier to read. I doubt a lot of merchants are using the snapshot builds since we never really get feedback from snapshot versions.